Documentation/using-rkt-with-systemd.md

```diff
@@ -9,7 +9,7 @@ In the shell excerpts below, a `#` prompt indicates commands that require root p
 The [`systemd-run`][systemd-run] utility is a convenient shortcut for testing a service before making it permanent in a unit file. To start a "daemonized" container that forks the container processes into the background, wrap the invocation of `rkt` with `systemd-run`:
 
 ```
-# systemd-run --slice=machine rkt run coreos.com/etcd:v2.2.5
+# systemd-run --slice=machine --service-type=notify rkt run coreos.com/etcd:v2.2.5
```

Comment:
`--service-type=notify` only applies to applications which support `sd_notify(3)`. I think we should make this a separate, dedicated example, giving it a sentence like:

> If your application supports `sd_notify(3)`, and the stage1 image is using systemd, use the following command to have systemd notification enabled: